Index: src/main/java/fi/ylihallila/remote/commons/Roles.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/fi/ylihallila/remote/commons/Roles.java	(revision 9e02e490ecfd8ef80b63c494cfe09660fae4092b)
+++ src/main/java/fi/ylihallila/remote/commons/Roles.java	(date 1593810511095)
@@ -1,8 +1,10 @@
 package fi.ylihallila.remote.commons;
 
+import io.javalin.core.security.Role;
+
 import java.util.Set;
 
-public enum Roles {
+public enum Roles implements Role {
 
     ANYONE("Anyone"),
     ADMIN("Administrative tasks"),
